tests/suites/test_suite_pkcs7.data

```diff
@@ -0,0 +1,63 @@
+PKCS7 Signed Data Parse Pass SHA256 #1
+depends_on:MBEDTLS_PKCS7_C
+pkcs7_parse:"data_files/pkcs7_data_cert.signed.sha256"
+
```

Comment:
`test_suite_pkcs7` fails in some configurations enumerated by `tests/scripts/depends-{pkalgs,hashes}.pl`. This indicates missing dependencies in test cases. At least some dependencies on `MBEDTLS_SHA1_C` and `MBEDTLS_PKCS1_V15` are missing.

Please prefer SHA-2 to SHA-1 in new test data.